doc/en/user/source/styling/sld/extensions/rendering-selection.rst

```diff
@@ -0,0 +1,306 @@
+.. _rendering_selection:
+
+Rendering Selection
+====================
+
+GeoServer provides a ``VendorOptions`` to define whether a particular element ``Rule``, ``FeatureTypeStyle`` or ``Symbolizer`` should be applied to a ``getLegendGraphic`` output or to a ``getMap`` output.
+The option is named ``inclusion`` and can take the following values: ``normal`` if no selection will apply, ``mapOnly`` if the SLD element should be used only during maps rendering and ``legendOnly`` if the element will be applied only when rendering legends eg.:
+
+* - <VendorOption name="inclusion">legendOnly</VendorOption>
+
+* - <VendorOption name="inclusion">mapOnly</VendorOption>
+
+If the value is set to ``legendOnly`` the element will be skipped when applying the style to the data to render map.
+If the value is set to ``mapOnly`` the element will be skipped when applying the style to the data to render legend.
+If the value is set to ``normal`` will have the same effect then omitting the VendorOption: the SLD element will be used for both map and legend.
```

Comment:
Looks like a great use case for a bullet list.

```suggestion
Valid values are:
-  ``legendOnly`` the element will be skipped when applying the style to the data to render map.
- ``mapOnly`` the element will be skipped when applying the style to the data to render legend.
- ``normal`` will have the same effect then omitting the VendorOption: the SLD element will be used for both map and legend (same effect as not specifying the vendor option).
```

Reply:
fixed